recipes/master.rb

```diff
@@ -87,6 +98,28 @@
   end
 end
 
+# salt-api default user acl
+# TODO: to be replaced by LWRP `external_auth`
+default_user_acl = {
```

Comment:
This should be part of `node['salt']['master']['config']` and not broken out into a separate file